tests/topotests/lib/common_config.py

```diff
@@ -3748,7 +3754,7 @@ def verify_cli_json(tgen, input_dict):
     return True
 
 
-@retry(attempts=3, wait=4, return_is_str=True)
+@retry(attempts=2, wait=4, return_is_str=True, initial_wait=2)
```

Comment:
Why are we changing evpn tests for pim?  Please go through this commit and fix it

Reply:
I have aligned with base changes